change/@fluentui-react-tag-picker-93adf770-e205-4d89-9513-308f47d37af6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32285&commitId=53c0cac16713832b89edfa2def69249a14bad2d2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|54| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.Full Overlay RTL.chromium.png|1139| Changed|
|Drawer.overlay drawer full.chromium.png|3315| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>